app/_hub/kong-inc/acme/index.md

```diff
@@ -16,6 +16,7 @@ categories:
 kong_version_compatibility:
     community_edition:
       compatible:
+        - 2.1.x
```

Comment:
This one is getting updated in https://github.com/Kong/docs.konghq.com/pull/2141